### PR TITLE
Prepare 2021 online edition

### PR DESCRIPTION
**Pull request by abelsromero**

### CODE REVIEW

**Review by anyulled — APPROVED** [review]

**Review by anyulled — APPROVED** [review]

**Review by alexdeveng — APPROVED** [review]

**Review by icougil — CHANGES_REQUESTED** [review]
Few things I found...

File: 2021/_data/homepage.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
   "og_title":"JBCNConf - The big JVM conference in Spain",
```
Comment (anyulled) [inline 2021/_data/homepage.json:2]:
```suggestion
  "og_title":"JBCNConf - The biggest JVM conference in Barcelona",
```

File: 2021/_data/menu.json [hunk 2]
```diff
@@ -4,6 +4,10 @@
       "label": "Home",
       "href": "/#JBCN"
     },
+    {
+      "label": "Call For Papers",
+      "href": "/allForPapers.html"
```
Comment (anyulled) [inline 2021/_data/menu.json:9]:
```suggestion
      "href": "/callForPapers.html"
```

**Review by anyulled — CHANGES_REQUESTED** [review]
minor suggestions

File: 2021/_data/homepage.json (hunk 1 above)
Comment (abelsromero) [reply 2021/_data/homepage.json:2]: Not exactly the same, plus is this kind of a slogan we have been using. I'd rather keep it as it is now

File: 2021/_data/homepage.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
-  "og_description": "12th to 14th of July 2021, Barcelona will be the host for three days to share knowledge and experiences, meet enthusiasts and geeks and learn about new technologies.",
+  "og_description": "23th of July 2021, Barcelona will be the host for three days to share knowledge and experiences, meet enthusiasts and geeks and learn about new technologies.",
```
[the remark this answers is not in the document]
Comment (abelsromero) [reply 2021/_data/homepage.json:3]: Simplified, thanks

File: 2021/_data/homepage.json [hunk 4]
```diff
@@ -49,15 +49,15 @@
...
-  "cfp_closing": "6th of May (last day)",
-  "talks-workshops": "Monday-Wednesday",
-  "talks_days_cfp": "Monday and Tuesday",
-  "talks_days": "Monday 12th & Tuesday 14th of July 2021",
-  "talks_days_from_to": "Monday 12th to Tuesday 13th",
-  "workshop_day_cfp": "Wednesday",
+  "days": "23",
+  "from-to": "23th of July",
+  "from_to_days": "23th of July 2021",
+  "cfp_closing": "18th of July (last day)",
```
[the remark this answers is not in the document]
Comment (abelsromero) [reply 2021/_data/homepage.json:55]: debería ser june, gracias

File: 2021/_data/homepage.json [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
-  "ticket_section_desc": "Do not miss the opportunity to participate in this event and to immerse yourself in 3-days interesting talks on Java, JVM and open-source technologies.",
-  "ticket_buy": "BUY YOUR TICKET!",
+  "ticket_section_desc": "Do not miss the opportunity to participate in this event and to immerse yourself in 1 day of interesting talks on Java, JVM and open-source technologies.",
+  "ticket_buy": "GET YOUR FREE TICKET!",
   "ticket_open": "1st of December 2020",
   "ticket_soon": "SOON",
   "ticket_soldout": "SOLD OUT",
   "ticket_staytuned" : "Stay tuned !!",
-  "tickets_url": "https://tickets.jbcnconf.com",
+  "tickets_url": "https://attendify.co/jbcnconf-2021---virtual-edition-tHubMQF",
```
Comment (icougil) [inline 2021/_data/homepage.json:12]:
```suggestion
  "tickets_url": "https://hopin.com/events/jbcnconf-2021-virtual/",
```

**Review by icougil — CHANGES_REQUESTED** [review]
Correct URL for tickets

**Pull request merged**